CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 #### Breaking Changes
 
+- TraceMultiPropagator extension has been removed. Users should migrate to the instance returned 
```

Comment:
this should probably move down a section about extensions rather than the API. And this is an enhancement to the API, rather than a breaking change, I think.

Reply:
We talked about this and agreed to let the release manager work it out.  😉